pages/dkp/konvoy/2.2/image-builder/override-files/default-or-files/index.md

```diff
@@ -10,9 +10,49 @@ menuWeight: 0
 
 Konvoy comes with default override files:
 
-- [FIPS](../../../fips/) override file.
-- [Nvidia](../../../choose-infrastructure/aws/gpu/) override file.
-- Offline override file.
-- Offline for FIPS override file.
+- FIPS override:
+  ```yaml
+  ---
+  k8s_image_registry: docker.io/mesosphere
 
-You can find these override files in the [Konvoy Image Builder repo](https://github.com/mesosphere/konvoy-image-builder/tree/main/overrides).
+  fips:
+  enabled: true
+
+  build_name_extra: -fips
+  kubernetes_build_metadata: fips.0
+  default_image_repo: hub.docker.io/mesosphere
+  kubernetes_rpm_repository_url: "https://packages.d2iq.com/konvoy/stable/linux/repos/el/kubernetes-v{{ kubernetes_version }}-fips/x86_64"
+  docker_rpm_repository_url: "\
+    https://containerd-fips.s3.us-east-2.amazonaws.com\
+    /{{ ansible_distribution_major_version|int }}\
+    /x86_64"
+  ```
+
+- Nvidia override:
+
+  ```yaml
+  ---
+  gpu:
+    types:
+      - nvidia
+
+  build_name_extra: "-nvidia"
+  nvidia_cuda_version: "470"
+  ```
+
+- Offline override:
+
+  ```yaml  
+  os_packages_local_bundle_file: "{{ playbook_dir }}/../artifacts/{{ kubernetes_version }}_{{ ansible_distribution|lower }}_{{ ansible_distribution_major_version }}_x86_64.tar.gz"
+  pip_packages_local_bundle_file: "{{ playbook_dir }}/../artifacts/pip-packages.tar.gz"
+  images_local_bundle_dir: "{{ playbook_dir}}/../artifacts/images"
+  ```
+
+- Offline for FIPS override:
```

Comment:
In the preview, this section appears blank: http://docs-d2iq-com-pr-4495.s3-website-us-west-2.amazonaws.com/dkp/konvoy/2.2/image-builder/override-files/default-or-files/